src/Components/WebAssembly/Sdk/integrationtests/WasmBuildIntegrationTest.cs

```diff
@@ -281,7 +281,7 @@ public async Task Build_Hosted_Works()
             Assert.FileDoesNotExist(result, buildOutputDirectory, "wwwroot", "_framework", "_bin", "blazorwasm.dll");
 
             var staticWebAssets = Assert.FileExists(result, buildOutputDirectory, "blazorhosted.StaticWebAssets.xml");
-            Assert.FileContains(result, staticWebAssets, Path.Combine("net5.0", "wwwroot"));
+            Assert.FileContains(result, staticWebAssets, Path.Combine("net6.0", "wwwroot"));
```

Comment:
My suggestion for test code is to compile https://github.com/dotnet/aspnetcore/blob/master/src/Framework/test/TestDataAttribute.cs into every test project and unconditionally include the bit at https://github.com/dotnet/aspnetcore/blob/master/src/Framework/test/Microsoft.AspNetCore.App.UnitTests.csproj#L19-L22. The code to read that attribute value is pretty simple but could also be centralized. Again, this will make it much easier to change next time around.